fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/CreateTableCommand.java

```diff
@@ -139,6 +139,8 @@ public void run(ConnectContext ctx, StmtExecutor executor) throws Exception {
                             // so we replace it to varchar
                             dataType = TypeCoercionUtils.replaceSpecifiedType(dataType,
                                     StringType.class, VarcharType.MAX_VARCHAR_TYPE);
```

Comment:
remove this? since we replace all CharacterType to VarcharType.MAX_VARCHAR_TYPE in next line